indigo/distribution.yaml

```diff
@@ -7393,11 +7393,11 @@ repositories:
   microstrain_3dm_gx5_45:
     doc:
       type: git
-      url: https://github.com/bsb808/microstrain_3dm_gx5_45.git
+      url: https://github.com/ros-drivers/microstrain_mips
```

Comment:
It looks like CI is unhappy because this is a URL to a webpage instead of the git repo URL. I think this should be `https://github.com/ros-drivers/microstrain_mips.git` instead, and same everywhere else in this PR.

Reply:
My apologies.  I just pushed a new commit which I believe should resolve the mistake.